File: Week_4/14.5_problem_1-loop_practice_1.py
# Problem Statement: Take a positive integer N as input and print the pattern shown in the sample input output.

""" sample input: 5

output:
    *
   **
  ***
 ****
***** 

"""


n = int(input())

s = n - 1
t = 1

for i in range(1, n + 1):
    for j in range(1, s + 1):
        print(" ", end="")
    for k in range(1, t + 1):
        print("*", end="")

    s -= 1
    t += 1
    
    print()

File: Week_4/14.5_problem_1-loop_practice_1.cpp
// Problem_1 Statement: Take a positive integer N as input and print the pattern shown in the sample input output.

/* 
sample input: 5

output:
    *
   **
  ***
 ****
*****

 */


#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;

    int s = n - 1;
    int t = 1;

    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= s; j++)
        {
            cout << " ";
        }
        for (int k = 1; k <= t; k++)
        {
            cout << "*";
        }
        s--;
        t++;
        cout << endl;
    }

    return 0;
}
